### DISCUSSION TITLE
Does Sidestore work with wlan and no internet ?

### DISCUSSION THREAD

**Discussion by wilhelm115** [SideStore/SideStore#610] [Q&A] [answered]
When I have mobile internet and a wifi with no internet, is it possible to refresh certificates ?

As far I understand sidestore emulates a mac to ask for renewal and the emu mac does the job on my phone (i guess it need internet for that) 
Could this connection be routed over my mobile connection (sidestore -> wlan -> vpn -> emulated a sidestore-mac -> cellular connection -> certificate server) ?

**Comment by polymo1** [comment]
SideStore MUST be used with a functional WiFi connection. It must be a wifi connection because the WireGuard loopback VPN only works correctly on a wifi connection, and it needs stable internet because it needs to reach the Anisette server for authentication.

**Reply by wilhelm115** [reply]
Hi, it's a functional WiFi but it has only Ethernet no Internet. Wouldn't it be possible to reach the Anisette server over mobile connection or route it over the VPN ?

**Reply by SternXD** [reply, marked as answer]
~~I mean it wouldn't hurt to try but we don't support it.~~ I didn't understand at first but no you can't do that
